### PR TITLE
Add bindings for plRandomSoundMod and plRandomCommandMod.

### PR DESCRIPTION
**Pull request by Deledrius**

### CODE REVIEW

File: Python/Module.cpp [hunk 1]
```diff
@@ -141,6 +141,9 @@
 #include "PRP/Modifier/pyOneShotMod.h"
 #include "PRP/Modifier/pyPostEffectMod.h"
 #include "PRP/Modifier/pyPythonFileMod.h"
+#include "PRP/Modifier/pyRandomCommandMod.h"
+#include "PRP/Modifier/pyRandomSoundMod.h"
+#include "PRP/Modifier/pyResponderModifier.h"
```
Comment (zrax) [inline Python/Module.cpp:146]: Duplicate include

File: core/PRP/Modifier/plRandomSoundMod.h [hunk 2]
```diff
@@ -49,6 +49,13 @@ class PLASMA_DLL plRandomSoundMod : public plRandomCommandMod
 protected:
     void IPrcWrite(pfPrcHelper* prc) HS_OVERRIDE;
     void IPrcParse(const pfPrcTag* tag, plResManager* mgr) HS_OVERRIDE;
+
+public:
+    const std::vector<plRandomSoundModGroup>& getGroups() const { return fGroups; }
+    std::vector<plRandomSoundModGroup>& getGroups() { return fGroups; }
+    void addGroup(const plRandomSoundModGroup& group) { fGroups.push_back(group); }
```
Comment (zrax) [inline core/PRP/Modifier/plRandomSoundMod.h:56]: Pass by value + `emplace_back` preferred

File: Python/PRP/Modifier/pyRandomSoundMod.cpp [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+#include <PRP/Modifier/plRandomSoundMod.h>
+#include "pyModifier.h"
+#include "PRP/pyCreatable.h"
+
+
+/* plRandomSoundModGroup */
+PY_PLASMA_NEW(RandomSoundModGroup, plRandomSoundModGroup)
+
+PY_PLASMA_TYPE(RandomSoundModGroup, plRandomSoundModGroup, "plRandomSoundModGroup wrapper")
+
```
Comment (zrax) [inline Python/PRP/Modifier/pyRandomSoundMod.cpp:29]: Why no getters and setters for plRandomSoundModGroup?

File: Python/PRP/Modifier/pyRandomSoundMod.cpp [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    pyRandomSoundMod_Type.tp_methods = pyRandomSoundMod_Methods;
+    pyRandomSoundMod_Type.tp_getset = pyRandomSoundMod_GetSet;
+    if (PyType_CheckAndReady(&pyRandomSoundMod_Type) < 0)
+        return nullptr;
+
+    Py_INCREF(&pyRandomSoundMod_Type);
+    return (PyObject*)&pyRandomSoundMod_Type;
+}
+
+PY_PLASMA_IFC_METHODS(RandomSoundMod, plRandomSoundMod)
```
Comment (zrax) [inline Python/PRP/Modifier/pyRandomSoundMod.cpp:127]: Newline at EOF please

File: Python/Module.cpp (hunk 1 above)
Comment (Deledrius) [reply Python/Module.cpp:146]: Whoops!

File: Python/PRP/Modifier/pyRandomSoundMod.cpp (hunk 4 above)
Comment (Deledrius) [reply Python/PRP/Modifier/pyRandomSoundMod.cpp:127]: Ugh, I saw that too, but between preparing the PR and submitting it I got interrupted and forgot to go back and fix it.

File: Python/PRP/Modifier/pyRandomSoundMod.cpp (hunk 3 above)
Comment (Deledrius) [reply Python/PRP/Modifier/pyRandomSoundMod.cpp:29]: None exist, not even in Plasma: https://github.com/H-uru/Plasma/blob/master/Sources/Plasma/FeatureLib/pfAudio/plRandomSoundMod.h#L50-L65

Comment (Hoikas) [reply Python/PRP/Modifier/pyRandomSoundMod.cpp:29]: The fields are public in Plasma. We will need to be able to use these fields from Python in order to properly support kickable SFX.

Comment (Deledrius) [reply Python/PRP/Modifier/pyRandomSoundMod.cpp:29]: Well, gosh.

File: core/PRP/Modifier/plRandomSoundMod.h [hunk 5]
```diff
@@ -32,6 +32,12 @@ class PLASMA_DLL plRandomSoundModGroup
     void write(hsStream* S);
     void prcWrite(pfPrcHelper* prc);
     void prcParse(const pfPrcTag* tag);
+
+    const std::vector<unsigned short>& getIndices() const { return fIndices; }
+    std::vector<unsigned short> getIndices() { return fIndices; }
```
Comment (zrax) [inline core/PRP/Modifier/plRandomSoundMod.h:37]:
```suggestion
    std::vector<unsigned short>& getIndices() { return fIndices; }
```

**Review by zrax — APPROVED** [review]

**Pull request merged**